pac4j-core/src/main/java/org/pac4j/core/context/HttpConstants.java

```diff
@@ -42,6 +42,8 @@
 
     String AUTHORIZATION_HEADER = "Authorization";
 
+    String BASIC_HEADER_PREFIX = "Basic ";
```

Comment:
Do we need to check for case? BASIC? basic? Is "Basic" pretty much the standard?